src/MongoDB.Driver.Legacy/CommandResults/MapReduceResult.cs

```diff
@@ -100,6 +102,7 @@ public long EmitCount
         /// <summary>
         /// Gets the output count.
         /// </summary>
+        [Obsolete("OutputCount is ignored by server versions 4.3.1 and newer.")]
```

Comment:
Slight rewording requested:

     [Obsolete("OutputCount is not available on server versions 4.3.1 and newer.")]

Reply:
done